### PR TITLE
updating mgt-people-picker enums with lowercase

### PR DESCRIPTION
**Pull request by vogtn**
updates enums with lowercase

### CODE REVIEW

File: concepts/toolkit/components/people-picker.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ By default, the `mgt-people-picker` component fetches people from the `/me/peopl
 | -------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | show-max | showMax   | A number value to indicate the maximum number of people to show. the default value is 6.                                                                                             |
 | group    | group     | A string value that belongs to a Microsoft Graph defined group for further filtering of the search results.                                                                            |
```
Comment (nmetulev) [inline concepts/toolkit/components/people-picker.md:27]: I think this should be group-id and groupId, do you mind updating it too?

File: concepts/toolkit/components/people-picker.md [hunk 2]
```diff
@@ -25,8 +25,8 @@ By default, the `mgt-people-picker` component fetches people from the `/me/peopl
...
-| type     | type      | The type of entities to search for. Available options are: `Person`, `Group`, `Any`. Default value is `Person`. This attribute has no effect if `group` property is set.                                                                            |
-| group-type     | groupType      | The group type to search for. Available options are: `Unified`, `Security`, `MailEnabledSecurity`, `Distribution`, `Any`. Default value is `Any`. This attribute has no effect if the `type` property is set to `Person`.                                                                           |
+| type     | type      | The type of entities to search for. Available options are: `person`, `group`, `any`. Default value is `person`. This attribute has no effect if `group` property is set.                                                                            |
```
Comment (nmetulev) [inline concepts/toolkit/components/people-picker.md:28]: same here, should be "if `group-id` attribute is set"

File: concepts/toolkit/components/people-picker.md (hunk 1 above)
Comment (vogtn) [reply concepts/toolkit/components/people-picker.md]: updated: 
b46e8d6

File: concepts/toolkit/components/people-picker.md (hunk 2 above)
Comment (vogtn) [reply concepts/toolkit/components/people-picker.md]: updated: 
b46e8d6

**Pull request merged**